frontend/src/scenes/funnels/funnelLogic.js

```diff
@@ -0,0 +1,72 @@
+import { kea } from 'kea'
+import api from 'lib/api'
+import { toParams } from 'lib/utils';
+
+
+
+export const funnelLogic = kea({
+    key: props => props.id,
+    actions: () => ({
+        setFilters: input => input
+    }),
+    loaders: ({ props, values }) => ({
+        funnel: {
+            loadFunnel: async (id = props.id) => {
+                return await api.get('api/funnel/' + id + '/?exclude_count=1')
+            },
+        },
+        steps: {
+            loadSteps: async (id = props.id) => {
+                return await api.get('api/funnel/' + id + '/?' + toParams(values.filters))
+            },
+        },
+        people: {
+            loadPeople: async (steps) => {
+                return await api.get('api/person/?id=' + steps[0].people.slice(0, 99).join(','))
+            }
+        },
+    }),
+    reducers: ({ actions }) => ({
+        filters: [
+            {},
+            {
+                [actions.setFilters]: (state, input) => {
+                    return { ...state, ...input }
+                },
+            }
+        ],
+        steps: [
+            null,
+            {
+                [actions.loadStepsSuccess]: (_, { steps }) => {
+                    return steps.steps
+                }
+            }
+        ],
+        people: [
+            null,
+            {
+                [actions.loadPeopleSuccess]: (_, { people }) => {
+                    console.log('loadingpeople', people)
+                    return people.results
+                }
+            }
+        ],
```

Comment:
These `steps` and `people` reducers are not needed, they are created automatically by loaders. You can just delete this code.